pandas/tests/series/test_missing.py

```diff
@@ -1521,12 +1521,23 @@ def test_interp_nonmono_raise(self):
             s.interpolate(method="krogh")
 
     @td.skip_if_no_scipy
-    def test_interp_datetime64(self):
+    @pytest.mark.parametrize("method", ["nearest", "pad"])
```

Comment:
you can parametrize on tz_naive_fixture here (and just remove the bottom test); this includes None and a bunch of timezones.